SharpDenizenTools/MetaObjects/MetaProperty.cs

```diff
@@ -100,8 +113,17 @@ public override bool ApplyValue(MetaDocs docs, string key, string value)
                 case "description":
                     Description = value;
                     return true;
-                case "example":
```

Comment:
shouldn't break examples, these were fine as-is

Reply:
No longer relevant(?) ref discussion on Discord

Reply:
should still leave the shared examples imo and concat them, to only separate when explicitly intended

Reply:
Added back the `@example` key and made it add the example into both lists - that way you can combine examples for both with examples for specific ones